setup.py

```diff
@@ -58,6 +58,7 @@
         'requests',
         'docopt',
         'publicsuffix',
+        'dnspython',
         'py3dns',
```

Comment:
Do we want to remove `py3dns` from the requirements?

Reply:
I did try removing `py3dns`, but then I got errors regarding a missing import that `spf` uses.  So I put it back in.